package.json

```diff
@@ -39,22 +40,22 @@
     "preversion": "npm test"
   },
   "dependencies": {
-    "@slack/client": "^4.2.0",
+    "@slack/client": "^4.2.2",
     "axios": "^0.18.0",
-    "body-parser": "^1.18.2",
+    "body-parser": "^1.18.3",
     "camel-case": "^3.0.0",
-    "chalk": "~2.3.2",
+    "chalk": "~2.4.1",
```

Comment:
any idea why we used `~` for chalk?

Reply:
For old version flowtype, so just use `^` instead.